plugin/trino-openlineage/src/main/java/io/trino/plugin/openlineage/OpenLineageListener.java

```diff
@@ -313,7 +313,7 @@ private List<OutputDataset> buildOutputs(QueryIOMetadata ioMetadata)
                             .namespace(this.datasetNamespace)
                             .name(getDatasetName(outputMetadata.getCatalogName(), outputMetadata.getSchema(), outputMetadata.getTable()))
                             .facets(openLineage.newDatasetFacetsBuilder()
-                                    .columnLineage(openLineage.newColumnLineageDatasetFacet(columnLineageDatasetFacetFieldsBuilder.build()))
+                                    .columnLineage(openLineage.newColumnLineageDatasetFacet(columnLineageDatasetFacetFieldsBuilder.build(), List.of()))
```

Comment:
When should we pass a non-empty list as the 2nd argument?

Reply:
I don't know how to compute input columns in that case.